### ISSUE TITLE
How to test the model on customized images

### ISSUE THREAD

**Issue by JanineCHEN** [LymanSong/FP_GNN#1]
Hi,

First of all, thanks for the great work! I am wondering how to apply the trained models on any customized raster images. Any help would be greatly appreciated!

**Comment by JanineCHEN** [comment]
I have constructed an evaluator and somehow get the models work on customizes images. Yet I found that the "cubicasa" checkpoints could not work on customized images , only the "UOS" and the "UOS_aug" ones can be loaded. Any idea why?

**Comment by LymanSong** [comment]
Hi, Thank you for having an interest in my work.

About the question, you need to pre-process and vectorize your
custom images before feeding them into neural networks. So you need to do
as follows,

1. save your own image dataset in ./dataset/your_dataset_name/fps

2. run main code with giving arguments like $ python main --dataset
your_dataset_name --

3. Then dataset_module will automatically make your own dataset instance by
processing raster images, vectorizing, and making graphs.

4. With the dataset instance, you can train your own models or predict
labels of polygons with pre-trained models.

But the pre-trained models are highly dependent on the respective dataset,
so you would not get a good result from them if you want to apply the
pre-trained models to your own fp images.

If Cubicasa pre-trained model is not loaded, there must be some problem
with your directories. The code will load pre-trained models
from ./checkpoint/'dataset argument', so you need to train your own model
with your dataset or save your own images in ./dataset/cubicasa/fps if you
want to use models trained on cubicasa for your own images.

It would not take much time to train if you have labels.

There must be many problems with my codes, so if you have more to ask, feel
free to send me an email.

Thank you

2021년 2월 7일 (일) 오후 8:02, JanineCHEN <notifications@github.com>님이 작성:
